WordPress/Classes/Utility/ContextManager.m

```diff
@@ -346,12 +342,14 @@ - (void)migrateDataModelsIfNecessary
         DDLogWarn(@"Migration required for persistent store.");
         NSError *error = nil;
         NSArray *sortedModelNames = [self sortedModelNames];
-        BOOL migrateResult = [ALIterativeMigrator iterativeMigrateURL:[self storeURL]
```

Comment:
I guess we can remove `ALIterativeMigrator` from the project.

Reply:
👍